examples/examples/tower-load.rs

```diff
@@ -1,410 +1,379 @@
+    }
+}
+
+impl std::error::Error for WrongMethod {}
+
+fn gen_uri(authority: &str) -> (usize, String) {
+    static ALPHABET: &str = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ";
+    let mut rng = rand::thread_rng();
+    let idx = rng.gen_range(0, ALPHABET.len() + 1);
+    let len = rng.gen_range(0, 26);
+    let letter = ALPHABET.get(idx..=idx).unwrap_or("");
+    (len, format!("http://{}/{}", authority, letter))
+}
+
+// .instrument(tracing::info_span!(target: "gen", "load_gen", remote.addr=%addr));
+#[tracing::instrument]
+async fn load_gen(addr: &SocketAddr) -> Result<(), Err> {
+    let svc = ServiceBuilder::new()
+        .buffer(5)
+        .layer(request_span::layer(req_span))
+        .timeout(Duration::from_millis(200))
+        .service(Client::new());
+
+    while let Some(_) = time::interval(Duration::from_millis(50)).next().await {
+        let authority = format!("{}", addr);
+        let mut svc = svc.clone();
+        svc.ready().await?;
+        let f = async move {
+            let sleep = rand::thread_rng().gen_range(0, 25);
+            time::delay_for(Duration::from_millis(sleep)).await;
+
+            let (len, uri) = gen_uri(&authority);
+            let req = Request::get(&uri[..])
+                .header("Content-Length", len)
+                .body(Body::empty())
+                .unwrap();
+
+            let rsp = match svc.call(req).await {
+                Err(e) => {
+                    error!(target: "gen", message = "request error!", error = %e);
+                    return Err(e);
+                }
+                Ok(rsp) => rsp,
+            };
+
+            let status = rsp.status();
+            if status != StatusCode::OK {
+                error!(target: "gen", message = "error received from server!", ?status);
+            }
+
+            let body = match hyper::body::to_bytes(rsp).await {
+                Err(e) => {
+                    error!(target: "gen", message = "body error!", error = ?e);
+                    return Err(e.into());
+                }
+                Ok(body) => body,
+            };
+            let body = String::from_utf8(body.to_vec())?;
+            info!(target: "gen", message = "response complete.", rsp.body = %body);
+            Ok(())
+        }
+        .instrument(span!(target: "gen", Level::INFO, "load_gen", remote.addr=%addr));
+        tokio::spawn(f);
+    }
+
+    Ok(())
+}
+
+fn req_span<A>(req: &Request<A>) -> Span {
+    let span = tracing::span!(
+        target: "gen",
+        Level::INFO,
+        "request",
+        req.method = ?req.method(),
+        req.path = ?req.uri().path(),
+        headers = ?req.headers()
+    );
+
+    {
+        // TODO: this is a workaround because tracing_subscriber::fmt::Layer
+        // doesn't honor overridden span parents.
+        let _enter = span.enter();
+        debug!(message = "received request.", req.headers = ?req.headers(), req.version = ?req.version());
```

Comment:
nit: this (and everything else in this file) uses the old-timey less idiomatic `message = "..."`, can we update these to use the more idiomatic style?

Reply:
At least with the example on 352, the non old-timey variant fails to compile with:

```rust
error: local ambiguity: multiple parsing options: built-in NTs tt ('field') or 1 other option.
   --> examples/examples/tower-load.rs:352:37
    |
352 |             info!(target: "gen", rsp.body = ?body, "response complete.");
    |                                     ^

error: aborting due to previous error

error: could not compile `examples`.

To learn more, run the command again with --verbose.
```

I'll switch over all the other options that _don't_ have this issue.